pkg/cmd/registry/artifact/crud/create/create.go

```diff
@@ -129,7 +129,7 @@ rhoas service-registry artifact create --type=JSON my-artifact.json
 	}
 
 	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", "json", opts.localizer.MustLocalize("registry.cmd.flag.output.description"))
-	cmd.Flags().StringVar(&opts.file, "file", "", "File location of the artifact")
+	cmd.Flags().StringVar(&opts.file, "output-file", "", "File location of the artifact")
```

Comment:
This works more like an input file I guess, wdyt?
Same query for `artifact update`.

Reply:
Sorry, yes you are right. this is for input 🤦🏻

Reply:
Please "Request changes" on this PR to prevent it from being merged.